mitmproxy/proxy/root_context.py

```diff
@@ -132,7 +132,6 @@ def __repr__(self):
 
 
 class Log(object):
-
```

Comment:
Lets fix this once and for all with a style enforcer. The specific choices don't matter as much as consistency.